numpy/linalg/linalg.py

```diff
@@ -2610,12 +2610,12 @@ def norm(x, ord=None, axis=None, keepdims=False):
 
 # multi_dot
 
-def _multidot_dispatcher(arrays):
+def _multidot_dispatcher(arrays, out=None):
     return arrays
 
 
 @array_function_dispatch(_multidot_dispatcher)
-def multi_dot(arrays):
+def multi_dot(arrays, out=None):
```

Comment:
Thoughts on making this as follows?
```suggestion
def multi_dot(arrays, *, out=None):
```

Reply:
+1 let's make this keyword only!

Reply:
+1 from me as well.

normally I'd be concerned about consistency between functions, but for `out` it seems like such an improvement to have keyword-only that we should just do it.

Reply:
@eric-wieser @shoyer @rgommers Good point. Added in 1f2697495f778ca3b552345250dc57d94b758cfd